webnode/src/components/consent-overlay/button-group/index.js

```diff
@@ -1,118 +1,88 @@
 import React from "react";
-import Radium from "radium";
-import color from "color";
+import styled from "styled-components";
 
-class Button extends React.Component {
-  render() {
-    const { onClick, kind, children } = this.props;
-    return (
-      <button onClick={onClick} style={[styles.base, styles[kind]]}>
-        {children}
-      </button>
-    );
+const Button = styled.button`
+  border-radius: 12px;
+  border: none;
+  cursor: pointer;
+  font-family: Poppins;
```

Comment:
how do you feel about short hand, we can combine these three properties? `font: font-style font-variant font-weight font-size/line-height font-family;`